www/js/lib/zimDirEntry.js

```diff
@@ -54,7 +54,7 @@ define([], function() {
         this.redirectTarget = dirEntryData.redirectTarget;
         this.cluster = dirEntryData.cluster;
         this.blob = dirEntryData.blob;
-        this.url = dirEntryData.url;
+        this.url = dirEntryData.namespace + '/' + dirEntryData.url;
```

Comment:
@peter-x  : I know this is already merged, but are you sure it's a good idea to modify this.url in the constructor?
Shouldn't we keep it as dirEntrydata.url?
I find it a bit confusing that one property 'url' might sometimes contain the namespace, and sometimes not.
And I think I already saw cases where a URL had two namespaces, like A/A/name.html (I'll have to check)